composer/yamls/models/deeplabv3_ade20k.yaml

```diff
@@ -0,0 +1,56 @@
+train_dataset:
+  ade20k:
+    datadir: null
+    split: train
+    base_size: 512
+    final_size: 512
+    drop_last: true
+    shuffle: true
+val_dataset:
+  ade20k:
+    datadir: null
+    split: val
+    base_size: 512
+    final_size: 512
+    drop_last: false
+    shuffle: false
+optimizer:
+  decoupled_sgdw:
+    lr: 0.04
+    momentum: 0.9
+    weight_decay: 5.0e-5
+    dampening: 0
+    nesterov: false
+schedulers:
+  - cosine_decay:
+      T_max: "127ep"
+      eta_min: 0
+      verbose: false
+      interval: step
+model:
+  deeplabv3:
+    initializers:
```

Comment:
Are these initializers being used?

Reply:
Ah, no... The original PyTorch DeepLabV3 did not initialize the head, so I was hesitant. I the initialization is being used now!